app/controllers/rooms_controller.rb

```diff
@@ -25,6 +25,10 @@ def create
 
   def update
     respond_to do |format|
+      if params[:room][:remove_image] == "1"
```

Comment:
🙋 Is this a Rails-y way of doing this?

Reply:
I would probably have a `Room::ImagesController` that has a `#destroy` action; and then I would send `DELETE /rooms/:room_id/images/:image_id` if a `Room` has_many `Images`. If it's a `has_one`, I would send a `DELETE /rooms/:room_id/image`